recipe/common.php

```diff
@@ -119,6 +119,7 @@
 option('tag', null, InputOption::VALUE_OPTIONAL, 'Tag to deploy');
 option('revision', null, InputOption::VALUE_OPTIONAL, 'Revision to deploy');
 option('branch', null, InputOption::VALUE_OPTIONAL, 'Branch to deploy');
+option('no-hooks', null, InputOption::VALUE_NONE, 'Run task without after/before hooks');
```

Comment:
Only this one. Other used by common recipe. So if your deploy.php does not require common.php, i may no need in tag/rev/branch options.